chkder.c

```diff
@@ -2,8 +2,8 @@
 #include <math.h>
 #include "cminpackP.h"
 
-#define log10e 0.43429448190325182765
```

Comment:
I'm not sure about that, but this macro could have side effects.
You should put parentheses around all these macros to avoid side effects.
For example, this should be:
`#define log10e ((real)0.43429448190325182765)`
same for all others

Reply:
sure

Reply:
also use the longer version for this one (for long double):
0.434294481903251827651128918916605082

Reply:
done